### PR TITLE
Megan tutorial

### PR DESCRIPTION
**Pull request by mes712**

### CODE REVIEW

File: portfolio/src/main/webapp/baking.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <title> Chef Megan</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1>Food Gallery:</h1>
+      <div class="macaron-thumbnails">
+          <p> Macarons: </p>
```
Comment (linhnguye-g) [inline portfolio/src/main/webapp/baking.html]: super nit: please remove space before Macarons

File: portfolio/src/main/webapp/baking.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+          <img src="/images/raz_mac.jpg" height=250/>
+          <img src="/images/rv_mac.jpg" height=250/>
+
+          <p>Not macarons:</p>
+          <img src="/images/biscotti.jpg" height=250/>
+          <img src="/images/cheese_tart.jpg" height=250/>
+          <img src="/images/crepe_cake.jpg" height=250/>
+          <img src="/images/challah.jpg" height=250/>
+          <img src="/images/pizza.jpg" height=250/>
+          <img src="/images/pasta.jpg" height=250/>
```
Comment (linhnguye-g) [inline portfolio/src/main/webapp/baking.html]: img tag ends with `>` only please remove the `/`
also please add an alt for all images

https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -6,22 +6,19 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+<body>
```
Comment (linhnguye-g) [inline portfolio/src/main/webapp/index.html]: body tag is misaligned. please align by indenting body tag over by two spaces!

https://google.github.io/styleguide/htmlcssguide.html#Indentation

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -6,22 +6,19 @@
...
     <div id="content">
       <h1>Megan Shi</h1>
       <img src="images/me.JPG" height=250/>
```
Comment (linhnguye-g) [inline portfolio/src/main/webapp/index.html]: img tag ends with `>` only please remove the `/`
also please add an alt for all images

https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -6,22 +6,19 @@
...
+          I like playing tennis with my mom and video games with my friends. I'm looking forward to learning
           about software engineering and building projects. I also hope to build a PC this 
           summer so if you have any guidance let me know :^)</p>
-
       <p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <p> Learn more about me by clicking the links below: </p>
+      <p><a href="baking.html">Macarons and Things<a/> -- A gallery of my best baking accomplishments</p>
```
Comment (linhnguye-g) [inline portfolio/src/main/webapp/index.html]: nit: Ending tag should be `</a>` not `<a/>`

File: portfolio/src/main/webapp/baking.html [hunk 6]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title> Chef Megan</title>
```
Comment (linhnguye-g) [inline portfolio/src/main/webapp/baking.html]: super nit: please remove space before Chef

**Review by linhnguye-g — CHANGES_REQUESTED** [review]
Nice job Megan. All the food you've made looks delicious!

Also consider using an html validator can help catch some of these errors: https://google.github.io/styleguide/htmlcssguide.html#HTML_Validity

File: portfolio/src/main/webapp/baking.html (hunk 2 above)
Comment (linhnguye-g) [inline portfolio/src/main/webapp/baking.html]: nit: Indent by 2 spaces at a time. Don’t use tabs or mix tabs and spaces for indentation.
https://google.github.io/styleguide/htmlcssguide.html#Indentation

I think these are indented by 4 spaces, please change to 2 to be consistent with rest of the file

**Review by christophervo-g — COMMENTED** [review]
Looks mostly good! Don't have much to add outside of the few style nits that Linh pointed out :)

**Review by linhnguye-g — APPROVED** [review]
Looks great!

**Pull request merged**